app/mailers/article_mailer.rb

```diff
@@ -0,0 +1,19 @@
+class ArticleMailer < ActionMailer::Base
+
+  def email_last_articles(user,total_articles)
+  	@articles=Article.last_created(total_articles)
+    mail :subject => "Here's what you missed!",
+         :to      => user.email,
+         :from    => Rails.application.secrets.credentials_user
+  end
+
+  def email_last_users_articles(user_email,total_users,total_articles_per_user)
+  	@users=User.last_created(total_users)
```

Comment:
This logic shouldnt be in a mailer. The mailer the only thing that does is a fetch of sth and render an html